test/ruleLoaderTests.ts

```diff
@@ -17,8 +17,7 @@
 import {loadRules} from "./lint";
 
 describe("Rule Loader", () => {
-    const path = require("path");
-    const rulesDirectory = path.join(global.process.cwd(), "build/rules");
+    const rulesDirectory = "build/src/rules";
```

Comment:
why is the old code not necessary?
this should probably be `RULES_DIRECTORY` now